### PR TITLE
[IMP] accounting: update doc about Purchase Receipts

### PR DESCRIPTION
**Pull request by jcs-odoo**
+ rename bills_or_receipts.rst -> purchase_receipts.rst

### CODE REVIEW

File: accounting/payables/supplier_bills/purchase_receipts.rst [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+details, and clicking on *Validate*.
+
+Edit the Journal Entry before posting it
+========================================
+
+Once you have filled out the *Invoice Lines* tab, you can directly reconcile the payment by
+modifying the **Journal Entry**.
+
+To do so, click on the **Journal Items** tab, and change the accounts according to your needs, then
+click on *Post*.
```
Comment (Lau-Odoo) [inline accounting/payables/supplier_bills/purchase_receipts.rst:37]:
```suggestion
To do so, click on the **Journal Items** tab, change the accounts according to your needs, and
click on *Post*.
```

File: accounting/payables/supplier_bills/purchase_receipts.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+=================
+Purchase Receipts
+=================
+
+**Purchase Receipts** are not invoices but rather confirmations of received payments, such as a
+ticket or a receipt.
+
+This feature is meant to be used when you pay directly with your company's money for an expense.
+**Vendor Bills**, on the other hand, are recorded when an invoice is issued to you and that the
+amount is first credited on a debt account before a later payment reconciliation.
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply accounting/payables/supplier_bills/purchase_receipts.rst:10]: I'd rather leave the first sentence alone. --> lead paragraph (= chapeau)

File: accounting/payables/supplier_bills/purchase_receipts.rst [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+To record a new receipt, go to :menuselection:`Accounting --> Vendors --> Receipts`, click on
+*Create*, fill out the form, and click on *Post*.
+
+.. image:: media/purchase_receipts_draft.png
+   :align: center
+   :alt: Fill out the draft purchase receipt in Odoo Accounting
+
+You can now register the payment by clicking on *Register Payment*, then filling out the payment's
+details, and clicking on *Validate*.
```
[the remark this answers is not in the document]
Comment (Lau-Odoo) [reply accounting/payables/supplier_bills/purchase_receipts.rst:28]: Nothing about this suggestion?

[the remark this answers is not in the document]
Comment (jcs-odoo) [reply accounting/payables/supplier_bills/purchase_receipts.rst]: I didn't see it. Apologies.

**Pull request merged**